### PR TITLE
#4: Removed RLS references in code.

### LINKED ISSUE #4: Remove RLS-specific parts after port
## Situation

Exasol Virtual Schema Lua started as a port of RLS Lua. Simply put it is RLS minus the row-restriction logic. In this issue we will remove the RLS-specific parts of the ported software.

## Acceptance Criteria

1. RLS logic removed
2. RLS mentions in documentation removed

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #4.

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          java-version: 11
+          cache: 'maven'
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v3
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Install Lua ${{ matrix.lua_version }}
+        uses: leafo/gh-actions-lua@v8.0.0
```
Comment (ckunki) [inline .github/workflows/ci-build.yml:37]: There is also 9.1.0 available.
Do we want to upgrade?

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        with:
+          luaVersion: ${{ matrix.lua_version }}
+      - name: Install LuaRocks
+        uses: leafo/gh-actions-luarocks@v4
+        with:
+          luarocksVersion: "3.8.0"
```
Comment (ckunki) [inline .github/workflows/ci-build.yml:43]: latest: 4.3.0?

**Review by kaklakariada — APPROVED** [review]

File: .github/workflows/ci-build.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        run: |
+          mvn --batch-mode org.sonarsource.scanner.maven:sonar-maven-plugin:sonar \
+              -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+              -DtrimStackTrace=false \
+              -Dsonar.organization=exasol \
+              -Dsonar.host.url=https://sonarcloud.io \
+              -Dsonar.login=$SONAR_TOKEN
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (ckunki) [inline .github/workflows/ci-build.yml:70]: identical to other projects -- probably fine

**Review by ckunki — COMMENTED** [review]
Could you please check the version I commented?

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (redcatbear) [reply .github/workflows/ci-build.yml:37]: Thanks for the tip. In the next PR.

File: .github/workflows/ci-build.yml (hunk 2 above)
Comment (redcatbear) [reply .github/workflows/ci-build.yml:43]: Next PR.

**Pull request merged**